CHANGELOG.md

```diff
@@ -35,24 +49,65 @@ See the docs for all the details.
     * [default-features](https://opensource.axo.dev/cargo-dist/book/config.html#default-features)
     * [precise-builds](https://opensource.axo.dev/cargo-dist/book/config.html#precise-builds)
 
+### Tag Formats
 
-## Fixes
+cargo-dist's git tag parser has been made much more robust and permissive:
+
+* You can now prefix release tags with anything preceding a '/'
+* The 'v' prefix on a version is now optional
+* You can now use package-name/v1.0.0 instead of package-name-v1.0.0
+
+Putting this all together, all of these formats are now allowed:
+
+* unified (release everything with the given version)
+  * v1.0.0
+  * 1.0.0
+  * blah/blah/v1.0.0
+  * whatever/1.0.0
+* precise (release only this one package)
+  * package-name-v1.0.0
+  * package-name-1.0.0
+  * package-name/v1.0.0
+  * package-name/1.0.0
+  * blah/blah/package-name/v1.0.0
+  * blah/blah/package-name/1.0.0
+  * blah/blah/package-name-v1.0.0
+  * blah/blah/package-name-1.0.0
 
-### Installation instructions
+And of course `-prerelease.1`-style suffixes can be added to any of those.
 
-* [Updated and corrected Arch Linux installation instructions](https://github.com/axodotdev/cargo-dist/pull/326).
+Thanks to @Sharparam for all the great work on the implementation and docs for this!
 
-### Manifest
+* @Sharparam + @gankra [impl](https://github.com/axodotdev/cargo-dist/pull/346)
+* [docs](https://opensource.axo.dev/cargo-dist/book/workspace-guide.html#announcement-tags)
 
-* [Fixes handling repository URLs that end in .git](https://github.com/axodotdev/cargo-dist/pull/298).
+### Bring Your Own Github Release
+
+A new `create-release` config has been added, which makes cargo-dist interoperate with things like
+[release drafter](https://github.com/release-drafter/release-drafter/) which create a draft body/title
+for your Github Release.
+
+When you set `create-release = false` cargo-dist will assume a draft Github Release for the current git tag already exists with the title/body you want, and just upload artifacts to it. At the end of a successful publish it will undraft the Github Release.
+
+* @gankra [impl](https://github.com/axodotdev/cargo-dist/pull/367)
+* [docs](https://opensource.axo.dev/cargo-dist/book/config.html#create-release)
+
+
+
+## Fixes
 
+* @mistydemeo [Fix a typo in deprecated rustup update lines](https://github.com/axodotdev/cargo-dist/pull/342)
+* @mistydemeo [Restore table headings in generated github release](https://github.com/axodotdev/cargo-dist/pull/357)
```

Comment:
Is this a fix or a minor feature?

Reply:
Yeah sorry I tossed it there because I had no docs to link / nothing insightful to say. If you do have some interesting thoughts on it, would love to have your amendments to this PR to add them.